doc/source/user_guide_embedding/index.rst

```diff
@@ -74,33 +74,49 @@ Mechanical's command line interface. For instance, the consider the following Py
   ns = DataModel.Project.Model.AddNamedSelection()
   ns.Name = "Jarvis"
 
-The above code can be written as a Python file, such as "file.py" with only the following content:
+The above code can be written as a Python file, such as ``file.py`` with only the following content:
 
 .. code:: python
 
   ns = DataModel.Project.Model.AddNamedSelection()
   ns.Name = "Jarvis"
 
-That python file does not contain the PyMechanical import statements, and can inside Mechanical using the command line
+Because the file does not contain the PyMechanical import statements, you can run
+``file.py`` using the command line inside Mechanical.
 
-**On Windows**
+**Using command line interface (CLI)**
 
-Open a command prompt and run this command:
+This can be achieved on both the Windows and Linux platforms using
+``ansys-mechanical`` cli from the virtual environment where ``ansys-mechanical-core``
+has been installed. Activate the virtual environment and then use CLI to run the scripts.
+If multiple Mechanical versions are installed in the same system,
+versions can be specified using ``-r`` flag. Use ``-h`` for more information.
 
 .. code::
 
-    "C:/Program Files/ANSYS Inc/v231/aisol/bin/winx64/AnsysWBU.exe -DSApplet -AppModeMech -script file.py"
+    ansys-mechanical -i file.py
 
 .. note::
+
+   Alternately user can use below commands with
+   command prompt and terminal in Windows and Linux systems respectively.
```

Comment:
Alternately user can use the following commands in the command of the Windows platform and the terminal for Linux systems.

Reply:
I think I missed this, Will do this in next PR.